packages/web/src/app/shell/footer/footer.component.html

```diff
@@ -24,8 +24,7 @@
     <div class="links">
       @for (link of links; track link) {
         <a class="e-btn e-btn--icon e-btn--sm" [href]="link.url" target="_blank" rel="noopener">
-          <img class="default-img" [src]="link.defaultImgSrc" [alt]="link.alt" />
-          <img class="inverted-img" [src]="link.invertedImgSrc" [alt]="link.alt" />
+          <img [src]="link.imgSrc" [alt]="link.alt" />
```

Comment:
Simplified the implementation of the footer links, by using `filter: brightness(0);` to make the images black, instead of having two different images.